lib/virt_autotest/utils.pm

```diff
@@ -307,7 +307,9 @@ sub check_failures_in_journal {
             record_soft_failure("Found new failures: " . $failures . " please take actions as described in poo#151361.\n");
             record_info("Found failures in journal log", "Found new failures: " . $failures . " please take actions as described in poo#151361.\n", result => 'fail');
         }
-        script_run("rsync root\@$machine:$logfile $logfile", die_on_timeout => 0) if $machine ne 'localhost';
+        # ignore the attempt timing out with "timeout 20" which exits before
+        # the script_run internal timeout
+        script_run("timeout 20 rsync root\@$machine:$logfile $logfile") if $machine ne 'localhost';
```

Comment:
Should larger timeout value be used here ?

Reply:
@Julie-CAO

Reply:
I guess 20 seconds is enough to copy a text log file from a vm to the host, but I can't make sure for now as I fail to recall the detail.

Reply:
We can also use a higher number like 28 but that means we would only give 2s to the timeout command to handle the termination of rsync. In the end it probably does not really matter as the `timeout` command is just a best-effort approach to clean up rsync. Afterwards same as before `script_run` will time out after 30s and continue execution regardless of the return code.